apps/trackers/jira/query.py

```diff
@@ -12,6 +13,12 @@
 
 logger = logging.getLogger(__name__)
 
+# From sdengine/models.py (constant NAME_RE).
+# Restrict CVE ID to valid years (first one assigned was in 1999; hopefully we'll transcend
+# software vulnerabilities in year 3000+), and disallow ID of 0000 and leading zeros for 4+
+# IDs (e.g. -00001 is an invalid ID).
+CVE_NAME_RE = re.compile(r"CVE-(?:1999|2\d{3})-(?!0{4})(?:0\d{3}|[1-9]\d{3,})")
```

Comment:
You can import this from `osidb/validators.py`

Reply:
Good catch, thanks.